tideways-daemon/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+extra="${DAEMON_OPTIONS} --hostname=${HOSTNAME:-tideways}"
+extra="${extra} --env=${TIDEWAYS_ENV:-development}"
```

Comment:
The beautiful spelling would be

`extra+=" --env=${TIDEWAYS_ENV:-development}"`